python/flask/README.md

```diff
@@ -0,0 +1,48 @@
+# Flask Example for [getsentry](https://github.com/getsentry)
+
+![Sentry logo](_ReadMeImages/sentry-logo-black.png)
+
+### Table of Contents
+- [Introduction](#introduction)
+- [About This Demo](#about-this-demo)
+- [Installing Sentry](#installing-sentry)
+
+## Introduction
+
+[Sentry](https://sentry.io/welcome/) provides open source error tracking that shows you every crash in your stack as it happens, with the details needed to prioritize, identify, reproduce, and fix each issue. It also gives you information your support team can use to reach out to and help those affected and tools that let users send you feedback for peace of mind.
+
+Sentry was conceived in 2010 with a simple aim of illuminating production application issues. It started as a tiny bit of Open Source code, and has since expanded to an incredible team and hundreds of contributors, and now support all popular languages and platforms.
+
+Read about how Sentry came to be on [StackShare](https://stackshare.io/posts/founder-stories-how-sentry-built-their-open-source-service).
+
+## About This Demo
+This demo provides a basic Hello World example of adding Sentry to an app. To play with this demo, you'll need to create a Sentry account, and update line 7 of app.py with your <PUBLIC_DSN_KEY> and <PRIVATE_DSN_KEY>. You can find these under Settings > Client Keys in your account.
```

Comment:
Just tried to step through instructions without assuming anything. This step should specify "Project Settings" since Client Keys isn't reachable from User/Account/Organization settings.

... granted it'll probably just move again in sentry 9